modules/oidcsso/src/main/java/org/wso2/carbon/tomcat/oidcsso/extension/utils/exception/AuthenticationResponseException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.tomcat.oidcsso.extension.utils.exception;
+
+/**
+ * This class defines the exception type which is used in processing authentication response
+ */
+
```

Comment:
Above comments relating to the removing unnecessary empty lines also apply here. This Javadoc is bound to the class, so it should be visually attached to the class declaration.